cypress/integration/my-orcid-sidebar.e2e.js

```diff
@@ -0,0 +1,118 @@
+describe('My Orcid sidebar' + runInfo(), () => {
+  describe('Orcid id', () => {
+    it('display the users Orcid', () => {})
+    it('display url to navigate to the public page view', () => {})
+  })
+  describe('Emails' + runInfo(), () => {
+    it('display a user with only a primary email "unverified"', () => {})
+    it('display a user with only a primary email "verified"', () => {})
+    it('display multiple emails "unverified" and "verified"', () => {
+      // Expect changes to be display outside and inside of the modal
+    })
+    describe('Emails edit modals' + runInfo(), () => {
+      it('add additional emails and display those as "unverified" with default privacy', () => {
+        // Expect changes to be display outside and inside of the modal
```

Comment:
After the user clicks the "Add another email" URL at the bottom, a new empty input box is shown. 
while the new input box is empty or invalid should the "Add another email" URL be hidden, disable, or kept?

Reply:
I'd keep it there and active. If the user wishes to add multiple empty emails then they can. I'm assuming empty or invalid fields wouldn't be saved or would show an error on 'Save changes'?

Reply:
Yes we can handle that a similar thing to the "additional emails" on the register form